content/hi/docs/reference/glossary/cluster.md

```diff
@@ -0,0 +1,20 @@
+---
+title: क्लस्टर
+id: cluster
+full_link: 
+short_description: >
+   कार्यकर्ता मशीनों का एक सेट होता है, जिसे नोड्स कहा जाता है, जो कंटेनरीकृत अनुप्रयोगों को चलाता है। प्रत्येक क्लस्टर में कम से कम एक कार्यकर्ता नोड होता है।
```

Comment:
Please indent all these descriptions using one tab character instead of multiple space characters.